benefits/pension_scheme.md

```diff
@@ -12,7 +12,10 @@ We provide a pension scheme inline with the UK Government's rollout of automatic
 
 As your employer we will contribute towards your pension alongside your own contribution. 
 
-From April 2019 the minimum amount that you pay into your pension is **5%** and we pay **3%**—a total minimum contribution equivalent to **8%** of your salary. 
+From April 2019 the minimum amount that you pay into your pension is **5%** and we pay **3%**—a total minimum contribution equivalent to **8%** of your salary.The way our pensions was setup a few years ago means that the 3% pensions is based on qualifying earnings of £50k.
```

Comment:
```suggestion
From April 2019 the minimum amount that you pay into your pension is **5%** and we pay **3%**—a total minimum contribution equivalent to **8%** of your [qualitifying earnings](qualifying-earnings-band--upper-limit). The way our pensions was setup a few years ago means that the 3% pensions is based on qualifying earnings of £50k.
```

It's not 8% of the salary, it's 8% of the upper limit of the qualifying earnings band.

Reply:
I think it would be beneficial to the employee if a few typical scenarios can be written out. Perhaps with expected pension contributions at various salary bands across the company, showing the working.